metrics/access-logs/sniped/go.mod

```
@@ -0,0 +1,8 @@
+module sniped
+
+go 1.14
```

```suggestion
go 1.16
```

Unless there's a reason to use 1.14?